manifest.json

```diff
@@ -18,27 +15,26 @@
       "images": [
         {
           "sharedTags": {
-            "1.0.7": {},
+            "1.0.8": {},
             "1.0": {},
-            "lts": {}
+            "lts": {
+              "isUndocumented": true
+            }
           },
           "platforms": [
             {
               "dockerfile": "1.0/jessie/runtime",
               "os": "linux",
               "tags": {
-                "1.0.7-jessie": {}
+                "1.0.8-jessie": {}
               }
             },
             {
               "dockerfile": "1.0/nanoserver-sac2016/runtime",
               "os": "windows",
               "tags": {
-                "1.0.7-nanoserver-sac2016": {},
-                "1.0.7-nanoserver": {
-                  "isUndocumented": true
-                },
-                "1.0.7-nanoserver-$(nanoServerVersion)": {
+                "1.0.8-nanoserver-sac2016": {},
+                "1.0.8-nanoserver": {
```

Comment:
I think the undocumented tags like this should be dropped during this service event instead of rev'ing the version.  The reason they exist was to preserve backwards compatibility when we introduced the multi-arch tags.

Reply:
Would you include `2.0.3-nanoserver` in that? It was an undocumented tag that was rev-ed in https://github.com/aspnet/aspnet-docker/pull/328. We could manually drop it from docker hub and remove it during this PR

Reply:
Yes those tags could also be removed.  I think at this point waiting until the next release would be the correct process to follow.